test/server/chat-plugins/chat-monitor.js

```diff
@@ -57,7 +57,6 @@ describe('Chat monitor', () => {
 			this.user.joinRoom(this.room.roomid, this.connection);
 
 			this.parse = async function (message) {
-				Chat.loadPlugins();
```

Comment:
Maybe restore this?

Reply:
I already did locally, not my fault my wifi sucks!!!